### PR TITLE
Scaffolded react app

### PR DESCRIPTION
**Pull request by Joberg95**
#### [Scaffold React App](https://www.pivotaltracker.com/story/show/165001620)

As a development team,
In order to be able to start adding features,
We would like to have a basic application structure.

#### Description
- Setup application
- Clean up comments

### CODE REVIEW

**Review by yanqiuzhang — APPROVED** [review]

**Review by aleciabenjamin — APPROVED** [review]

File: src/index.js [hunk 1]
```diff
@@ -0,0 +1,8 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+import './index.css';
+import App from './App';
+import * as serviceWorker from './serviceWorker';
+
+ReactDOM.render(<App />, document.getElementById('root'));
```
Comment (tochman) [inline src/index.js:7]: And where exactly is the element with id `root` ? I can't find it in the `public/index.html` 😳

**Review by tochman — CHANGES_REQUESTED** [review]

File: src/App.js [hunk 2]
```diff
@@ -0,0 +1,28 @@
+import React, { Component } from 'react';
+import logo from './logo.svg';
```
Comment (aleciabenjamin) [inline src/App.js]: pls remove logo and app.css

File: README.md [hunk 3]
```diff
@@ -1 +1,2 @@
-Please stop messing up the repos :)
+## thyme
+I wont mess up this repo
```
Comment (aleciabenjamin) [inline README.md]: pls remove

File: src/App.js [hunk 4]
```diff
@@ -0,0 +1,28 @@
...
+import './App.css';
+
+class App extends Component {
+  render() {
+    return (
+      <div className="App">
```
Comment (tochman) [inline src/App.js]: where is the `App` class defined?

File: src/App.js [hunk 5]
```diff
@@ -0,0 +1,28 @@
...
+        <header className="App-header">
```
Comment (tochman) [inline src/App.js]: where is the `App-header` class defined?

File: src/App.js [hunk 6]
```diff
@@ -0,0 +1,28 @@
...
+          <img src={logo} className="App-logo" alt="logo" />
```
Comment (tochman) [inline src/App.js]: where is the `App-logo` class defined?

File: src/App.js [hunk 7]
```diff
@@ -0,0 +1,28 @@
+import React, { Component } from 'react';
+import logo from './logo.svg';
+import './App.css';
```
Comment (tochman) [inline src/App.js]: Can't see this file - `App.css`

File: src/App.js (hunk 2 above)
Comment (tochman) [inline src/App.js]: Can't see this file - `logo.svg`

File: src/App.js [hunk 8]
```diff
@@ -0,0 +1,28 @@
...
+  render() {
+    return (
+      <div className="App">
+        <header className="App-header">
+          <img src={logo} className="App-logo" alt="logo" />
+          <p>
+            Edit <code>src/App.js</code> and save to reload.
+          </p>
+          <a
+            className="App-link"
```
Comment (tochman) [inline src/App.js]: where is the `App-link` class defined?

File: src/index.js (hunk 1 above)
Comment (Joberg95) [reply src/index.js:7]: Added root id div back into html file

File: src/App.js (hunk 4 above)
Comment (Joberg95) [reply src/App.js]: removed

File: src/App.js (hunk 5 above)
Comment (Joberg95) [reply src/App.js]: removed

File: src/App.js (hunk 6 above)
Comment (Joberg95) [reply src/App.js]: removed

File: src/App.js (hunk 7 above)
Comment (Joberg95) [reply src/App.js]: deleted this file and then import

File: src/App.js (hunk 2 above)
Comment (Joberg95) [reply src/App.js]: logo import removed

File: src/App.js (hunk 8 above)
Comment (Joberg95) [reply src/App.js]: Removed

**Review by tochman — APPROVED** [review]

**Review by NiciP — APPROVED** [review]

**Review by yanqiuzhang — APPROVED** [review]

**Review by aleciabenjamin — APPROVED** [review]

**Pull request merged**